### PR TITLE
[zh] Sync IstioCon China 2023 blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync IstioCon China 2023 blog into Chinese:
```
content/zh/blog/2023/istiocon-china/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by wilsonwu** [comment]
> you can include this PR too, if it makes sense for the Chinese version: https://github.com/istio/istio.io/pull/13950/files

Got it, I will apply it when it merged.

**Comment by wilsonwu** [comment]
> > you can include this PR too, if it makes sense for the Chinese version: https://github.com/istio/istio.io/pull/13950/files
> 
> Got it, I will apply it when it merged.

Done

File: content/zh/blog/2023/istiocon-china/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+[Istio Day 欧洲站](/zh/blog/2023/istio-at-kubecon-eu/)于 4 月举行，
+[Istio Day 北美站](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/)将于今年 11 月举行。
+
+IstioCon 致力于打造行业领先的服务网格，
+提供一个平台来探索从现实世界的 Istio 部署中获得的见解、
+参与交路实践活动以及与整个 Istio 生态系统的维护人员进行联系。
+
+除了 [IstioCon 2023 虚拟](https://events.istio.io/)活动之外，
+[IstioCon 2023 中国站](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/co-located-events/istiocon-cn/)也于 9 月 26 日在中国上海举行。
+该活动是 KubeCon + CloudNativeCon + 中国开源峰会的一部分，
```
Comment (zirain) [inline content/zh/blog/2023/istiocon-china/index.md:20]: `KubeCon + CloudNativeCon + Open Source Summit China` 只翻译一部分让这句话显得更加怪异

File: content/zh/blog/2023/istiocon-china/index.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+---
+
+能够再次安全地聚在一起真是太好了。经过两年只举办虚拟活动后，
+我们已经填满了 2023 年的日历。
+[Istio Day 欧洲站](/zh/blog/2023/istio-at-kubecon-eu/)于 4 月举行，
+[Istio Day 北美站](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/)将于今年 11 月举行。
+
+IstioCon 致力于打造行业领先的服务网格，
+提供一个平台来探索从现实世界的 Istio 部署中获得的见解、
+参与交路实践活动以及与整个 Istio 生态系统的维护人员进行联系。
```
Comment (zirain) [inline content/zh/blog/2023/istiocon-china/index.md:16]: 这里的maintainers翻译成了维护人员，下文又翻译成了维护者，请尽量保持统一。

File: content/zh/blog/2023/istiocon-china/index.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+由 Istio 维护者和 CNCF 安排和主办。
+我们非常自豪能够在上海举办如此盛大的 IstioCon 活动，
+并很高兴将中国 Istio 社区的成员聚集在一起。
+此次活动证明了 Istio 在亚太生态系统中巨大的受欢迎程度。
+
+{{< image link="./group-pic.jpg"
+    caption="IstioCon 2023 中国站"
+    >}}
+
+IstioCon 中国站在程序委员会成员 Jimmy Song 和 Zhonghu Xu
```
Comment (zirain) [inline content/zh/blog/2023/istiocon-china/index.md:30]: 请将附上相关人员的中文姓名

File: content/zh/blog/2023/istiocon-china/index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+{{< image width="75%"
+    link="./ambient-l4.jpg"
+    caption="IstioCon 2023 中国站，深入研究 Istio 网络流程和配置以实现 Ambient 和 Sidecar 的共存"
+    >}}
+
+华为基于 eBPF 的新 Istio 数据平面打算在内核中实现
+L4 和 L7 的能力，避免内核态和用户态切换，降低数据平面的延迟。
+通过 Xie SongYang 和 Xu Zhonghu 的一段有趣的谈话解释了这一点。
+来自英特尔的 Chun Li 和 Iris Ding 还将 eBPF 与 Istio 集成，
+他们的演讲“在 Istio Ambient 模式中利用 eBPF 进行流量重定向”引发了更有趣的讨论。
```
Comment (zirain) [inline content/zh/blog/2023/istiocon-china/index.md:62]: 这里好像是英文符号？

File: content/zh/blog/2023/istiocon-china/index.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    link="./diamond-sponsor.jpg"
+    caption="IstioCon 2023，我们的钻石赞助商"
+    >}}
+
+最后但并非不重要的一点是，我们要感谢 IstioCon
+中国程序委员会成员的辛勤工作和支持！
+
+{{< image width="75%"
+    link="./istiocon-program-committee.jpg"
+    caption="IstioCon 2023 中国站，程序委员会成员（拍摄者：Iris Ding）"
```
Comment (zirain) [inline content/zh/blog/2023/istiocon-china/index.md:120]: 原文的意思应该是 Iris Ding 作为委员会成员并没有在图中

File: content/zh/blog/2023/istiocon-china/index.md (hunk 3 above)
Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md:30]: Jimmy Song 可以找到中文名字宋净超，Zhonghu Xu 搜遍全网没有找到中文名，不确定是哪几个具体的汉字，所以所有人的名字一律未翻译。

**Review by zirain — CHANGES_REQUESTED** [review]
尽量在翻译后，通读下全文，必要的时候可以意译。

File: content/zh/blog/2023/istiocon-china/index.md (hunk 3 above)
Comment (zirain) [reply content/zh/blog/2023/istiocon-china/index.md:30]: 徐中虎

**Comment by wilsonwu** [comment]
> 尽量在翻译后，通读下全文，必要的时候可以意译。

收到，太难了，有些 Reviewer 要求直译，有些要求意译，还是要每次 case by case 看。

File: content/zh/blog/2023/istiocon-china/index.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
+---
+title: "IstioCon 2023 中国站总结"
+description: 在上海 KubeCon + CloudNativeCon + 中国开源峰会上快速回顾 Istio。
+publishdate: 2023-09-29
+attribution: "IstioCon 2023 中国站程序委员会; Translated by Wilson Wu (DaoCloud)"
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+能够再次安全地聚在一起真是太好了。经过两年只举办虚拟活动后，
+我们已经填满了 2023 年的日历。
```
[the remark this answers is not in the document]
Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md:10]: Done, did some improvement.

File: content/zh/blog/2023/istiocon-china/index.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
+---
+title: "IstioCon 2023 中国站总结"
+description: 在上海 KubeCon + CloudNativeCon + 中国开源峰会上快速回顾 Istio。
```
Comment (windsonsea) [inline content/zh/blog/2023/istiocon-china/index.md]:
```suggestion
description: 简要回顾中国上海 KubeCon + CloudNativeCon + Open Source Summit 的 Istio 演讲。
```

File: content/zh/blog/2023/istiocon-china/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md:20]: Done

File: content/zh/blog/2023/istiocon-china/index.md (hunk 2 above)
Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md]: All Done

File: content/zh/blog/2023/istiocon-china/index.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
+---
+title: "IstioCon 2023 中国站总结"
```
Comment (windsonsea) [inline content/zh/blog/2023/istiocon-china/index.md:2]:
```suggestion
title: "IstioCon China 2023 总结"
```

**Comment by zirain** [comment]
> > 尽量在翻译后，通读下全文，必要的时候可以意译。
> 
> 收到，太难了，有些 Reviewer 要求直译，有些要求意译，还是要每次 case by case 看。

感谢您的付出，但是博客可能会被其他地方引用，所以可能审核的时候确实大家有不通的意见。

File: content/zh/blog/2023/istiocon-china/index.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+description: 在上海 KubeCon + CloudNativeCon + Open Source Summit China 上快速回顾 Istio。
```
Comment (windsonsea) [inline content/zh/blog/2023/istiocon-china/index.md:3]:
```suggestion
description: 简要回顾上海 KubeCon + CloudNativeCon + Open Source Summit China 中的 Istio 主题演讲。
```

File: content/zh/blog/2023/istiocon-china/index.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+---
+
+经过两年时间只举办虚拟活动后，能够再次线下欢聚一堂真是太好了。
+我们的活动已经排满了 2023 年的日历。
+[Istio Day 欧洲站](/zh/blog/2023/istio-at-kubecon-eu/)于 4 月举行，
+[Istio Day 北美站](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/)将于今年 11 月举行。
+
+IstioCon 致力于打造行业领先的服务网格，
+提供一个平台来探索从现实世界的 Istio 部署中获得的见解、
+参与交路实践活动以及与整个 Istio 生态系统的维护者进行联系。
```
Comment (zirain) [inline content/zh/blog/2023/istiocon-china/index.md:16]:
```suggestion
参与交互式实践活动，并与整个 Istio 生态系统的维护者进行交流。
```

File: content/zh/blog/2023/istiocon-china/index.md (hunk 3 above)
Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md:30]: Jimmy Song = 宋净超
Zhonghu Xu = 徐中虎
Huailong Zhang = ？
Yuxing Zeng = ？
Xie SongYang = ？
Chun Li = ？
Iris Ding = ？
Kebe Liu = 刘齐均
Xiaopeng Han = 韩小鹏
Chaomeng Zhang = ？
Xi Ning Wang = ？
Zehuan Shi = ？
Huabing Zhao = 赵化冰
John Zheng = ？

这些名字需要对应中文，如果有你知道的，麻烦帮忙补充一下，感谢！

**Review by windsonsea — COMMENTED** [review]
可以参考 Sched 日程：https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/program/schedule/

File: content/zh/blog/2023/istiocon-china/index.md (hunk 3 above)
Comment (windsonsea) [reply content/zh/blog/2023/istiocon-china/index.md:30]: 稍等我拉你进 istio 群，自己问吧 😅

Comment (zirain) [reply content/zh/blog/2023/istiocon-china/index.md:30]: 张怀龙
曾宇星
李纯
丁少君
张超盟
王夕宁
郑风

Xie SongYang
Zehuan Shi 
我再确认下

Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md:30]: 已在群众咨询，稍后更新，先改为 Draft。

Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md:30]: Jimmy Song = 宋净超
Zhonghu Xu = 徐中虎
Huailong Zhang = 张怀龙
Yuxing Zeng = 曾宇星
Xie SongYang = 谢颂杨
Chun Li = 李纯
Iris Ding = 丁少君
Kebe Liu = 刘齐均
Xiaopeng Han = 韩小鹏
Chaomeng Zhang = 张超盟
Xi Ning Wang = 王夕宁
Zehuan Shi = 史泽寰
Huabing Zhao = 赵化冰
John Zheng = 郑风
Qin Shilin = 覃士林

Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md]: All Done

**Review by zirain — COMMENTED** [review]
LGTM, @zhlsunshine @rootsongjc PTAL.

File: content/zh/blog/2023/istiocon-china/index.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+强调了 Ambient 模式在 Istio 社区中的重要性和优先级，
+特别是对于像 Google Cloud 这样的我们的顶级支持者。
+
+{{< image width="75%"
+    link="./sponsored-keynote-google.jpg"
+    caption="IstioCon 2023 中国站，Google Cloud 赞助的主题演讲"
+    >}}
+
+主题演讲结束后，英特尔的张怀龙和阿里巴巴的曾宇星讨论了
+Ambient 和 Sidecar 共存的配置：对于想要尝试新
```
Comment (zhlsunshine) [inline content/zh/blog/2023/istiocon-china/index.md:50]: Hi @wilsonwu, there is a relative blog: https://istio.io/latest/blog/2023/traffic-for-ambient-and-sidecar/, is it possible to link here?

File: content/zh/blog/2023/istiocon-china/index.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+提供一个平台来探索从现实世界的 Istio 部署中获得的见解、
+参与交互式实践活动，并与整个 Istio 生态系统的维护者进行交流。
+
+除了 [IstioCon 2023 虚拟](https://events.istio.io/)活动之外，
+[IstioCon 2023 中国站](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/co-located-events/istiocon-cn/)也于 9 月 26 日在中国上海举行。
+该活动是 KubeCon + CloudNativeCon + Open Source Summit China 的一部分，
+由 Istio 维护者和 CNCF 安排和主办。
+我们非常自豪能够在上海举办如此盛大的 IstioCon 活动，
+并很高兴将中国 Istio 社区的成员聚集在一起。
+此次活动证明了 Istio 在亚太生态系统中巨大的受欢迎程度。
```
Comment (zhlsunshine) [inline content/zh/blog/2023/istiocon-china/index.md:24]:
```suggestion
此次活动展示了 Istio 在亚太生态系统中深受开发者和用户的欢迎。
```

File: content/zh/blog/2023/istiocon-china/index.md (hunk 11 above)
Comment (wilsonwu) [reply content/zh/blog/2023/istiocon-china/index.md]: Done, added the English version blog link, I will update it to the Chinese version when it available in website.

File: content/zh/blog/2023/istiocon-china/index.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
+---
+title: "IstioCon China 2023 总结"
+description: 简要回顾上海 KubeCon + CloudNativeCon + Open Source Summit China 中的 Istio 主题演讲。
+publishdate: 2023-09-29
+attribution: "IstioCon 2023 中国站程序委员会; Translated by Wilson Wu (DaoCloud)"
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+经过两年时间只举办虚拟活动后，能够再次线下欢聚一堂真是太好了。
```
Comment (zhlsunshine) [inline content/zh/blog/2023/istiocon-china/index.md:9]:
```suggestion
经过两年时间只举办虚拟活动后，能够再次线下欢聚一堂真是太好了！
```

**Comment by zhlsunshine** [comment]
Hi @wilsonwu, LGTM, except 3 comments, nice work!

**Review by hanxiaop — APPROVED** [review]
Nice!

**Comment by wilsonwu** [comment]
For the lint limitation, just use the: https://istio.io/latest/blog/2023/traffic-for-ambient-and-sidecar/ link, I will update it to Chinese relative path.

**Comment by wilsonwu** [comment]
> For the lint limitation, just use the: https://istio.io/latest/blog/2023/traffic-for-ambient-and-sidecar/ link, I will update it to Chinese relative path later.

@zhlsunshine still can't pass the lint check, so I will remove the link now and add it back after the Chinese version of blog published.

**Comment by zirain** [comment]
you can do that with follow up PR

**Review by zirain — APPROVED** [review]

**Pull request merged**